striatum/storage/history.py

```diff
@@ -141,6 +142,8 @@ def add_history(self, context, action, reward=None):
                               reward_time, reward)
             self.histories[history.history_id] = history
         self.n_histories += 1
+        self.last_history_id += 1
+        return self.last_history_id
```

Comment:
simply return `history.history_id` and remove `last_history_id`. 
`self.n_histories` is doing the same thing as your `self.last_history_id`